config.py

```diff
@@ -19,26 +19,36 @@
 import traceback
 #from urlparse import urlparse
 
+# 3p
+import simplejson as json
+
 # project
-from util import check_yaml
+from util import check_yaml, config_to_yaml
 from utils.platform import Platform, get_os
 from utils.proxy import get_proxy
+from utils.sdk import load_manifest
 from utils.service_discovery.config import extract_agent_config
 from utils.service_discovery.config_stores import CONFIG_FROM_FILE, TRACE_CONFIG
 from utils.service_discovery.sd_backend import get_sd_backend, AUTO_CONFIG_DIR, SD_BACKENDS
 from utils.subprocess_output import (
     get_subprocess_output,
     SubprocessOutputEmptyError,
 )
+from utils.windows_configuration import get_registry_conf, get_windows_sdk_check
+
 
 # CONSTANTS
-AGENT_VERSION = "1.0.0"
 STACKSTATE_CONF = "stackstate.conf"
 UNIX_CONFIG_PATH = '/etc/sts-agent'
 MAC_CONFIG_PATH = '/opt/stackstate-agent/etc'
+AGENT_VERSION = "5.16.0"
```

Comment:
Do we have to change this to our version number?

Reply:
Made this a bogus number to not confuse the user